baseapp/options.go

```diff
@@ -35,3 +37,75 @@ func SetMinimumFees(minFees string) func(*BaseApp) {
 	}
 	return func(bap *BaseApp) { bap.SetMinimumFees(fees) }
 }
+
+// nolint - Setter functions
+func (app *BaseApp) SetName(name string) {
```

Comment:
Can we pretty please add new lines between the function definitions?

Reply:
The point of not doing that was to not have to write nolint a ton of times. If you prefer we can still change it, I don't have a preference

Reply:
Just add `// nolint` to the top of the file then.